storage/spanstore/utils.go

```diff
@@ -0,0 +1,17 @@
+package spanstore
```

Comment:
this file is missing license.

Reply:
not a big fan of the `utils.go` filename, it's a lazy catch all. We can call this hash.go